xml/Microsoft.Extensions.Hosting/BackgroundService.xml

```diff
@@ -109,7 +109,15 @@
         <summary>This method is called when the <see cref="T:Microsoft.Extensions.Hosting.IHostedService" /> starts. The implementation should return a task that represents
             the lifetime of the long running operation(s) being performed.</summary>
         <returns>A <see cref="T:System.Threading.Tasks.Task" /> that represents the long running operations.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+See [Worker Services in .NET](https://docs.microsoft.com/dotnet/core/extensions/workers) for implementation guidelines.
```

Comment:
@gewarren I think I remember seeing that URLs from this domain can get the domain truncated. Is it the case with this one?

Reply:
Nevermind, I see your review. GitHub fails to refresh the page or show comments when they are provided.